### ISSUE TITLE
autolinking not working for android 

### ISSUE THREAD

**Issue by kostyamarevtsev** [underscopeio/react-native-facebook-account-kit#196]
before command react-native link react-native-facebook-account-kit 

file MainApplication 

```
import android.app.Application;
import com.facebook.react.PackageList;
import com.facebook.react.ReactApplication;
import com.swmansion.gesturehandler.react.RNGestureHandlerPackage;
import io.underscope.react.fbak.RNAccountKitPackage;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.facebook.soloader.SoLoader;
....
    @Override
    protected List<ReactPackage> getPackages() {
      @SuppressWarnings("UnnecessaryLocalVariable")
      List<ReactPackage> packages = new PackageList(this).getPackages();
      // Packages that cannot be autolinked yet can be added manually here, for example:
      // packages.add(new YourPackageName());
      return packages;
    }
....
```

add manually dependencies 

```
    @Override
    protected List<ReactPackage> getPackages() {
      @SuppressWarnings("UnnecessaryLocalVariable")
      List<ReactPackage> packages = new PackageList(this).getPackages();
      // Packages that cannot be autolinked yet can be added manually here, for example:
        packages.add(new RNGestureHandlerPackage());
        packages.add(new RNAccountKitPackage());
      return packages;
    }
```

react-native run-android 

Result : BUILD FAILED 
`error: package android.support.v4.content does not exist`

For ios autolinking work without problem

### Additional Information

* React Native Account Kit version: 1.2.0
* React Native version: 0.60.0
* Platform(s) (iOS, Android, or both?): Android
* Platform Version(s): 9.0

**Comment by jpgarcia** [comment]
Hi @kostyamarevtsev thank you for reporting this. 

Currently the library doesn't support `0.60.x` there's an open PR #192 related to that but ir seems that is not backward compatible with previous versions of react-native. We need to realize how are we're going to handle this change but I couldn't find time to work on that yet.

Let us know if you have any suggestion!

**Comment by jpgarcia** [comment]
Hi @kostyamarevtsev we just released the new **2.0.0** version of the lib that supports **RN 0.60.0*
